source/index.html.md

```diff
@@ -167,6 +169,7 @@ video_duration | The length of the video in **seconds** [float]
 video_stream_type | 'live' or 'on-demand'
 video_producer | The producer of the video title
 video_encoding_variant | An optional detail that allows you to compare different encoding settings.
+video_cdn | The name of the CDN, for comparing across CDN providers. Possible values include 'akamai', 'fastly', 'cloudfront', 'limelight', 'level3', 'custom', 'none'. Contact mux support to include additional CDN providers.
```

Comment:
@jondahl just realized we still need to pull this into the docs. Want to review first or does it look good?